provision.sh

```diff
@@ -190,20 +200,26 @@ install_golang() {
 
 install_nodejs() {
   echo ">>> Installing NodeJS"
-  curl -fsSL https://deb.nodesource.com/setup_16.x | bash -
-  apt-get -y install nodejs
+  #curl -fsSL https://deb.nodesource.com/setup_16.x | bash -
+  #apt-get -y install nodejs
+  curl -sL https://nsolid-deb.nodesource.com/nsolid_setup_4.x | sudo bash -
+  apt-get -y install nsolid-gallium nsolid-console
```

Comment:
Did you face any issue with the standard nodejs? We are wondering if this would work in the gcloud machines so we would prefer sticking to the original unless it is necessary to change it.

Reply:
unfortunately, i don't recall the details, but yes - the installation failed with them.

Reply:
There was a deprecation warning:
>>> Installing NodeJS

================================================================================
▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓
================================================================================

                           SCRIPT DEPRECATION WARNING


  This script, located at https://deb.nodesource.com/setup_X, used to
  install Node.js is deprecated now and will eventually be made inactive.

  Please visit the NodeSource distributions Github and follow the
  instructions to migrate your repo.
  https://github.com/nodesource/distributions

  The NodeSource Node.js Linux distributions GitHub repository contains
  information about which versions of Node.js and which Linux distributions
  are supported and how to install it.
  https://github.com/nodesource/distributions


                          SCRIPT DEPRECATION WARNING

================================================================================
▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓
================================================================================

TO AVOID THIS WAIT MIGRATE THE SCRIPT
Continuing in 60 seconds (press Ctrl-C to abort) ...

Reply:
The deprecation warning says that one should migrate to [NodeSource distributions](https://github.com/nodesource/distributions). [`nsolid`](https://nodesource.com/products/nsolid) looks like a monitoring tool that maybe installs nodejs as a dependency. I doubt we need the monitoring capabilities, how about we just install it as per the [instructions](https://github.com/nodesource/distributions#nodejs)?

Reply:
makes sense, of course. Corrected.